index.html

```diff
@@ -25,7 +25,7 @@
         <section class="section-1">
             <h1>Iheagwara<br>Augustine</h1>
         <p>
-            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
 If you like what you see and have a project you need coded, don’t hesitate and contact me.
         </p>
         <button class="Start-collab">Start collaboration</button>
```

Comment:
- [ ] Kindly use `required` on your input elements to trigger validation when trying to submit the form. It will ensure an empty form is not submitted. 🥸️
- [ ] Kindly put `name` attribute on your input elements so the Formspree service can function correctly. 🤝️
This is what is getting displayed when trying to submit the form.

<img src="https://user-images.githubusercontent.com/17243325/205714908-6f810b4b-4699-4b72-9fc4-24bb44e2d48f.png" width="300">

You can start with
```
<input type="text" name="name"/>
<input type="email" name="email"/>
<textarea type="text" name="message"></textarea>
```